tests/bitcoin/BitcoinProcessor.spec.ts

```diff
@@ -309,13 +309,13 @@ describe('BitcoinProcessor', () => {
   describe('transactions', () => {
     it('should get transactions since genesis capped by page size in blocks', async (done) => {
       const verifyMock = spyOn(bitcoinProcessor, 'verifyBlock' as any).and.returnValue(Promise.resolve(true));
+      // return as many as page size
+      const transactions: TransactionModel[] = createTransactions(BitcoinProcessor['pageSizeInBlocks'], bitcoinProcessor['genesisBlockNumber'], true);
       bitcoinProcessor['lastProcessedBlock'] = {
```

Comment:
Sanity: the test is getting transaction since genesis, so shouldn't `lastProcessedBlock` be `undefined`?

Reply:
This is simulating the caller trying to get transactions since genesis (genesis to genesis + 99), and the bitcoin's last Processed block is genesis + 100